orderer/common/cluster/deliver_test.go

```diff
@@ -271,8 +271,9 @@ func (ds *deliverServer) addExpectProbeAssert() {
 
 func (ds *deliverServer) addExpectPullAssert(seq uint64) {
 	ds.seekAssertions <- func(info *orderer.SeekInfo, _ string) {
-		assert.NotNil(ds.t, info.GetStart().GetSpecified())
-		assert.Equal(ds.t, seq, info.GetStart().GetSpecified().Number)
+		seekSpecified := info.GetStart().GetSpecified()
+		assert.NotNil(ds.t, seekSpecified)
+		assert.Equal(ds.t, seq, seekSpecified.Number)
```

Comment:
how does this fix the nil pointer panic? If **seekSpecified** is nil then at line 276 it will still cause a panic.
We need to use **require** instead of **assert** to fail-fast.

Reply:
good catch, in the attached logs on the bug, it was panicking on the 2nd call to `info.GetStart().GetSpecified()` so I just assumed we didn't expect it to be nil on the first call before the nil check. I figured at the time if it only ever panicked on the 2nd call then caching the first call would be enough to handle the nil check but I do agree changing to `require` would handle a panic on either call.

Reply:
Why don't we just require that **info.GetStart()** won't be nil as well?